book/testing/README.md

```diff
@@ -266,32 +270,46 @@ Here's a simple example of a test written in this style.  While the
 test generates output (though a call to `print_endline`), that output
 isn't captured in the source, at least, not yet.
 
-<!-- TODO avsm: what's with open! used in this chapter. -->
-<!-- Rest of book uses `open` -->
-
 ```ocaml file=examples/erroneous/trivial_expect_test/test.ml
 open! Base
-open! Stdio
+open Stdio
 
 let%expect_test "trivial" =
   print_endline "Hello World!"
 ```
 
+::: {data-type=note}
+##### `open` and `open!`
+
+In this example, we use `open!` instead of `open` because we happen
+not to be usinging any values from `Base`, and so the compiler will
+warn us about an unused open.
+
+But because `Base` is effectively our standard library, we want to
+keep it open anyway, since we want any new code we write to find
+`Base`'s libraries, not the ordinary OCaml standard library.  The
```

Comment:
Perhaps refer to Base "modules" here, and not" libraries".  We refine that definition later in the platform chapter.